client/wordpress-com.js

```diff
@@ -314,54 +314,44 @@ if ( config.isEnabled( 'reader' ) ) {
 	} );
 }
 
-if ( config.isEnabled( 'vip' ) ) {
-	sections.push( {
-		name: 'vip',
-		paths: [ '/vip', '/vip/deploys', '/vip/billing', '/vip/support', '/vip/backups', '/vip/logs' ],
-		module: 'vip',
-		secondary: true
-	} );
-}
+sections.push( {
```

Comment:
@scottsweb It looks like you created scaffolding for VIP section. It doesn't look like it is actively developed. Do you use those pages or would it be good idea to remove those files?